website/src/app/docs/deploy/advanced/external-database/readme.mdx

```diff
@@ -17,15 +17,12 @@ database services like Amazon RDS. See the
 [configuration ](#configure-firezone-to-connect) section below for more
 information configuring Firezone with an external DB.
 
-<Alert
-  color="warning"
-  html={`
-Configuring Firezone to use an external database can be complicated and
-error-prone. We recommend using the bundled Postgres for Omnibus-based
-deployments or the official Postgres Docker image for Docker-based deployments
-if possible.
-`}
-/>
+<Alert color="warning">
+  Configuring Firezone to use an external database can be complicated and
```

Comment:
Correct, but this is for Firezone 0.7. These docs are around for legacy users and I didn't want to move them because SEO. If you view them, https://docs.firezone.dev, you'll see an alert at the top.

Maybe that needs to be more prominent?

Reply:
It's all right, it could be a little taller. I missed it since I was just looking at the diff